### DISCUSSION TITLE
Show x-axis labels in custom timezone

### DISCUSSION THREAD

**Discussion by VovkViktor** [apexcharts/apexcharts.js#3127] [Q&A]
Hello. I want to show x-axis labels in custom timezone as be in categories array, but Apexchart shows it in my local timezone. How Can I do that labels show in the needed timezone? 

datetimeUTC: false
my categories format: 
```
[
    "2022-05-04T00:00:00+04:00",
    "2022-05-04T01:00:00+04:00",
    "2022-05-04T02:00:00+04:00",
    "2022-05-04T03:00:00+04:00",
    "2022-05-04T04:00:00+04:00",
    "2022-05-04T05:00:00+04:00",
    "2022-05-04T06:00:00+04:00",
    "2022-05-04T07:00:00+04:00",
    "2022-05-04T08:00:00+04:00",
    "2022-05-04T09:00:00+04:00",
    "2022-05-04T10:00:00+04:00",
    "2022-05-04T11:00:00+04:00",
    "2022-05-04T12:00:00+04:00",
    "2022-05-04T13:00:00+04:00",
    "2022-05-04T14:00:00+04:00",
    "2022-05-04T15:00:00+04:00",
    "2022-05-04T16:00:00+04:00",
    "2022-05-04T17:00:00+04:00",
    "2022-05-04T18:00:00+04:00"
]
```
but charts is 
<img width="1327" alt="image" src="https://user-images.githubusercontent.com/92850433/166699798-348bdd37-c538-45e0-b524-1c986e374185.png">

**Comment by MarioPerini** [comment]
you can use formatter https://apexcharts.com/docs/options/xaxis/#formatter for the xAxis and formatter on tooltips https://apexcharts.com/docs/options/xaxis/#tooltipFormatter
